tmt/config/__init__.py

```diff
@@ -83,7 +83,9 @@ def last_run(self, workdir: Path) -> None:
                 "'tmt run --last' might not pick the right run directory."
             )
         except OSError as error:
-            raise tmt.utils.GeneralError(f"Unable to save last run '{self.path}'.\n{error}")
+            raise tmt.utils.GeneralError(
+                f"Unable to save last run '{self.path}'.\n{error}"
```

Comment:
Yes, as mentioned below, part of this PR is to get rid of errors being encoded in error messages in this 1:1 manner.